bin/convertToIcns

```diff
@@ -48,23 +47,12 @@ trap cleanUp EXIT
 mkdir -p "${TEMP_DIR}"
 mkdir -p "${ICONSET}"
 
-PNG_PATH="${TEMP_DIR}/icon.png"
-${CONVERT_TO_PNG} "${SOURCE}" "${PNG_PATH}"
+sips -s format tiff "$SOURCE" --out "$TEMP_DIR/icon_128.tiff" -z 128 128 2>&1 > /dev/null & 
```

Comment:
`--setProperty` should be set instead of `-s`. Short flags are great when typing a command, but are less useful in a script, where comprehension is more important than speed.

When you can understand what a command is doing without having to open the man page and check every flag because they consist of a single letter, you have a better experience, and might be more inclined to fix bugs if an outsider to the code.